app/controllers/passwords_controller.rb

```diff
@@ -6,7 +6,7 @@ class PasswordsController < Devise::RegistrationsController
 
   def update_password
     if resource.update_with_password(password_update_params)
-      bypass_sign_in resource, scope: :user
+      sign_out(resource)
```

Comment:
Is this the forgot password scenario? If it's changing the password from profile scenario, then we shouldn't sign out the user when password is updated. Bypass the sign in.

Ignore if both of us are on the same page.

Read: https://security.stackexchange.com/a/178669

Reply:
Then we should not sign out the user.

Reply:
@yedhink Should we use a field which the user can select to log out of all active sessions? 

Also, do you want me to make similar changes to the email form?

Reply:
@rishabhanand12 

>Should we use a field which the user can select to log out of all active sessions?

Let's take that up in another issue. Not in this one. Because we'd have to regenerate the auth token for the user for logging out of all devices.

>Also, do you want me to make similar changes to the email form?

No. Email form, we have to log the user out.